Mono.Debugging/Mono.Debugging.Evaluation/ObjectValueAdaptor.cs

```diff
@@ -785,8 +785,7 @@ public virtual CompletionData GetExpressionCompletionData (EvaluationContext ctx
 					}
 
 					// FIXME: handle types and namespaces...
-				} catch (Exception ex) {
-					ctx.WriteDebuggerError (ex);
+				} catch {
```

Comment:
It will just be spew elsewhere, this exceptions are legit and OK exception, but not when trying to get CodeCompletion, in that case it doesn't make sense to show them...